js/sdks/packages/cli/src/docker-compose/types.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2021 by LunaSec (owned by Refinery Labs, Inc)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { DevelopmentConfigOptions } from '../config/types';
+
+import { LunaSecStackEnvironment } from './constants';
+import { DefinitionsService } from './lib/docker-compose-types';
+
+export type LunaSecService =
+  | 'localstack'
+  | 'localstack-proxy'
+  | 'lunasec-cli'
+  | 'secure-frame-iframe'
+  | 'application-front-end'
+  | 'application-back-end'
+  | 'tokenizer-backend'
+  | 'integration-test'
+  | 'nginx-live-demo';
+
+export interface ComposeService {
+  name: string;
+  config: DefinitionsService | null;
+}
+
+export interface HealthCheckOptions {
+  endpoint?: string;
+  protocol?: string;
+  composeOptions?: {
+    test?: string[];
+    timeout?: string;
+    interval?: string;
+    retries?: number;
+  };
+}
+
+export interface DockerComposeImageConfig {
+  image: string;
+}
+
+export interface LunaSecServiceCreationConfig {
+  environmentConfig: DevelopmentConfigOptions;
+  env: LunaSecStackEnvironment;
+  localBuild: boolean;
+  getBaseServiceConfig(name: string, excludeEnv?: boolean): DefinitionsService;
+  getDockerImageName(tag: string): DockerComposeImageConfig;
+  getDockerfileTarget(dockerfile: string, target: string): DefinitionsService;
+}
+
+export interface LunaSecReactEnv {
+  REACT_APP_EXPRESS_URL: string;
+  REACT_APP_SIMPLE_TOKENIZER_URL: string;
+  REACT_APP_TOKENIZER_URL: string;
+  REACT_APP_GRAPHQL_URL: string;
+}
+
+export interface LunaSecDockerEnv {
```

Comment:
nice

Reply:
are these the vars consumed by the "dedicated tokenizer"? Or just used in multiple palces.  This group is just called "docker"